build/download7z.js

```diff
@@ -0,0 +1,83 @@
+const path = require("path");
+const fs = require("fs");
+const { spawnSync } = require("child_process");
+
+(async () => {
+  // node-fetchはESModuleなので、import()で読み込む
+  const { default: fetch } = await import("node-fetch");
+  const distPath = path.resolve(__dirname, "vendored", "7z");
+  let url;
+  let filesToExtract;
+  switch (process.platform) {
+    case "win32": {
+      const sevenzrUrl = "https://www.7-zip.org/a/7zr.exe";
```

Comment:
7zrと7zaの２つをダウンロードする理由が初見でわからないかもなので、コメントするのはどうでしょう。
```suggestion
      // Windows版7-Zipは7z形式で配布されているので、まず7zだけを解凍できる7zrを入手する
      const sevenzrUrl = "https://www.7-zip.org/a/7zr.exe";
```

Reply:
詳しく書いてみました。